README.md

```diff
@@ -0,0 +1,21 @@
+# Lockout
```

Comment:
I switched the text file to markdown, which is a simple way to do nice formatting. read more about it https://guides.github.com/features/mastering-markdown/